docs/guide/ecosystem/documentation.md

```diff
@@ -114,28 +128,18 @@ Since designers usually don't like to wreck JavaScript, storybook is the best ch
 - [storybook-addon-vue-info](https://github.com/pocka/storybook-addon-vue-info) detects props info
 - [storybook-vue-router](https://github.com/gvaldambrini/storybook-router) add routes to your stories
 
-### Vue Styleguidist
+---
 
-[Vue-styleguidist](https://github.com/vue-styleguidist/vue-styleguidist) is a port of [react-styleguidist](https://github.com/styleguidist/react-styleguidist) for vue components.
-If you are building a Design System and want to showcase the powers of your components, this would probably be your first choice.
-
-Example: [vue-styleguide](http://rafaelescala.com/vue-styleguide/) or [vue-design-system](https://vueds.com/example/)
+- Honorable mention to [vue-storybook](https://github.com/mattrothenberg/vue-storybook). In single file components, write stories in a custom `<story>` block!
 
-It supports all the following vue features:
+### Vue Styleguidist
 
-- Single file components
```

Comment:
Why did we drop the features? Those are actually very useful to compare with others?